Aspire BudgetingTests/MockSpreadsheet.swift

```diff
@@ -0,0 +1,30 @@
+//
+// MockSpreadsheet.swift
+// Aspire BudgetingTests
+//
+
+
+import Foundation
+import GoogleAPIClientForREST
+
+class MockSpreadsheet: GTLRSheets_Spreadsheet {
+  init(sheets: [GTLRSheets_Sheet]) {
+    super.init()
+    super.sheets = sheets
+  }
+
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+}
+
+class MockSheet: GTLRSheets_Sheet {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/c832bf4daa051d3d91d7e9434f5fff223299bc47/Aspire BudgetingTests/MockSpreadsheet.swift#L21"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>